mongo/options/findoptions.go

```diff
@@ -433,68 +433,89 @@ func MergeFindOneOptions(opts ...*FindOneOptions) *FindOneOptions {
 	return fo
 }
 
-// FindOneAndReplaceOptions represent all possible options to the FindOneAndReplace() function.
+// FindOneAndReplaceOptions represents options that can be used to configure a FindOneAndReplace instance.
```

Comment:
Consider for each of the `FindAnd..Options` structs linking to https://docs.mongodb.com/manual/reference/command/findAndModify/index.html.  It's the only place I can find a good description for some of the options, in particular for arrayFilters

Reply:
You probably link to that already for the helper methods, so if you don't want to duplicate the link that's ok.

Reply:
Yeah, it's already linked in the relevant methods, so I'd prefer not to duplicate.